Code/Tabs.cs

```diff
@@ -71,6 +72,10 @@ public override void _Ready()
 
     public void Add(int amount)
     {
+        //Cooking menu has no total tab yet. This prevents out of bounds exceptions
+        if (_selected == 3)
```

Comment:
Going to just keep this in. I don't have a very good idea on where to put the "total" since there is no food tab really